AWS-docker-compose.yml

```diff
@@ -26,6 +27,8 @@ services:
         logging:
             driver: awslogs
             options:
-                awslogs-region: ap-southeast-1
-                awslogs-group: meteor
+                #(BEGIN) HARDCODED variable
+                awslogs-region: ap-southeast-1      
```

Comment:
@tuananh1508 is it possible to use a varable instead instead of the hard coded version?

Reply:
Fixed in https://github.com/Unee-T-INS/frontend/pull/7/commits/9cf1fad916eb0293046067cb7016b7781795933e